### PR TITLE
Adds new style yml workflow example to README

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds an example for the new `yml` Action syntax. @jonrohan r?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,27 @@ action "Export SVG from Figma" {
 }
 ```
 
+**Example worklfow using yml**
```
Comment (jonrohan) [inline README.md]: I think we should replace the old "example" and use this one instead. Can you make that update @schalkneethling ?

**Review by jonrohan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (schalkneethling) [reply README.md]: Sure thing

**Comment by schalkneethling** [comment]
@jonrohan updated. I also updated the URL about `secrets` as the current one does not work as expected.

**Review by jonrohan — APPROVED** [review]
thanks @schalkneethling!

**Pull request merged**